### ISSUE TITLE
Equations do not match in SW2007

### ISSUE THREAD

**Issue by astonetiger** [JohannesPfeifer/DSGE_mod#6]
Dear Professor  Pfeifer ,
     I am replicating the SW2007, but I find some of my derivations are the same as the appendix provided by the authors but different from the ones in the paper. 
   Besides the issues that you listed in your dynare code,  there are 3 problems. In the first 2 problems, I could not understand how they are derived. According to the appendix and my derivations, there are no  (\phi_p-1)\epsilon_{p}+1 and (\phi_w-1)\epsilon_w+1.
 1)  Equation (10) New-Keynesian Phillips curve in the paper, the coefficient of \mu_t^p is \pi_3. There is an extra expression (\phi_p-1)\epsilon_{p}+1 in the paper, comparing with the equation (48) in the appendix. It is (cfc-1)*curvp+1) in  New Keynesian Phillips Curve in the code. 
2)  Similarly, equation (13) in the paper, there is an extra expression (\phi_w-1)\epsilon_w+1 in the paper, comparing with the equation (61) in the appendix. it is (clandaw-1)*curvw+1 in Wage Phillips Curve in the code.
3)  Are the coefficients of \epsilon_t^p and \epsilon_t^w (curvatures of the Kimball goods market aggregator and labor market aggregator) normalized to unity? they are not 1 in the appendix equation (48) and (61).

 I look forward to your reply, thank you for your help.

**Comment by JohannesPfeifer** [comment]
@astonetiger Did you solve the issue on your own?

**Comment by astonetiger** [comment]
Sorry.

I thought no one cared about these questions, and these questions might wast your time, so I closed this issue and deleted the problems.

I mailed Professor Wotuers to seek help. He told me that I should refer to the option 2 in the appendix for the prices, which was shown in the equation 53, not 48.  The option 1 could not result in the equation in the paper.  And for the rest questions,  I am right.

Thank you!



> 
> 
> @astonetiger Did you solve the issue on your own?
